README.md

```diff
@@ -1,2 +1,71 @@
-# step179-2020
-Capstone Project by Megan Shi, Megha Kak, and Kevin Shao.
\ No newline at end of file
+# ClubHub: Google Student Training in Engineering Program (STEP) Capstone Project
+Capstone Project by Megan Shi, Megha Kak, and Kevin Shao.
+
+## Overview
+The high-level goal of ClubHub is to  ClubHub is to ease access to resources and information provided 
+by student organizations on college campuses and their officers. Many student organizations face 
+difficulty in spreading awareness of their club’s goals and events they host. Frequently, officers 
+are required to post on multiple social media platforms, such as Facebook, Instagram, Gmail, 
+and many more in order to get a portion of their membership population to attend an event. 
+
+ClubHub provides a central location for all student organizations to be listed, and for students and club
+officers to interact smoothly. To learn about a club, or just to explore new clubs, students need only navigate 
+to the web app, sign in (with Google, of course), and get started!
+
+## Features
+1. Explore page
+
+The landing page for signed-in users is the explore page, where all clubs are listed. The explore page 
+provides several options for viewing the clubs (as they will surely grow over time). The user may sort 
+the clubs chronologically, alphabetically, and by size (number of members), as well as filtering by labels 
+which the clubs tag themselves with. By doing so, anyone can quickly and easily find a club they're looking for, 
+or a club they didn't even know existed. If still unsatisfied, of course they can create their own. 
+
+2. Club page
+
+The club page has 3 panels: a information page, an announcements page, and a calendar page. The information page 
+gives basic information about the clubs (description, size, labels, officers, etc.), and the announcements page shows 
+recent announcements made by the club. Officers, of course, may edit information, post and edit announcements as they please. 
```

Comment:
super nit: I would remove the "of course" -> "Officers may edit information, post and..."